sdks/python/apache_beam/internal/cloudpickle_pickler.py

```diff
@@ -113,7 +116,8 @@ def dumps(
     o,
     enable_trace=True,
     use_zlib=False,
-    enable_best_effort_determinism=False) -> bytes:
+    enable_best_effort_determinism=False,
```

Comment:
would it make sense to make     enable_best_effort_determinism part of the CloudPickleConfig?